telco-core/configuration/core-baseline.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: core-baseline-16
+policyDefaults:
+  namespace: ztp-core-policies
+  policySets: []
+  placement:
+    clusterSelectors:
+      common: "core"
+      version: "4.16"
```

Comment:
Ouch.. all the version specific field will make it a pain to branch. We will need a really good branching automation to make sure we won't miss these.

Reply:
Agreed... On the RAN DU side there is some build automation which updates these per release (we leave "latest" in the upstream). @lack do you know where that is done?

Reply:
Is it done via Prow? 
https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching